### PR TITLE
Feature/polygonal faceset texture

### PR DESCRIPTION
**Pull request by SergejMuhic**
1. Added IfcTextureCoordinateIndices and documentation
2. Adding IfcTextureCoordinateIndicesWithVoids and documentation
3. Adding attribute IfcIndexedPolygonalFace.HasTexCoords
4. Added schema references

### CODE REVIEW

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcPresentationAppearanceResource/Entities/IfcTextureCoordinateIndices/Documentation.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+The _IfcTextureCoordinateIndices_ provide the texture coordinates for an _IfcIndexedPolygonalFace_. The _TexCoordIndex_ holds a list of indices pointing into the _IfcTextureVertexList_ for texture coordinates that correspond to the _TexCoordsOf.CoordIndex holding a list of indices pointing into the _IfcCartesianPointList3D_ for vertex coordinates.
```
Comment (pjanck) [inline IFC4x3/Sections/Resource definition data schemas/Schemas/IfcPresentationAppearanceResource/Entities/IfcTextureCoordinateIndices/Documentation.md]:
```suggestion
The _IfcTextureCoordinateIndices_ provide the texture coordinates for an _IfcIndexedPolygonalFace_. The _TexCoordIndex_ holds a list of indices pointing into the _IfcTextureVertexList_ for texture coordinates that correspond to the _TexCoordsOf.CoordIndex_ holding a list of indices pointing into the _IfcCartesianPointList3D_ for vertex coordinates.
```

File: IFC4x3/Sections/Resource definition data schemas/Schemas/IfcPresentationAppearanceResource/Entities/IfcTextureCoordinateIndicesWithVoids/Documentation.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+The _IfcTextureCoordinateIndicesWithVoids_ is a subtype of _IfcTextureCoordinateIndices_ to be used to provide texture coordinates to polygonal faces with inner loops. The two dimensional list of _TexCoordIndex_ holds the indices into the _IfcTextureVertexList_ that correspond to the list of _CoordIndex_ at _TexCoordsOf_ where:
+
+- the first dimension corresponds to the list position of the inner loops at _IfcIndexedPolygonalFaceWithVoids_ referenced by _TexCoordsOf_
+- the second dimension corresponds to the list position of the vertices at the nth inner loop at _IfcIndexedPolygonalFaceWithVoids_
```
Comment (pjanck) [inline IFC4x3/Sections/Resource definition data schemas/Schemas/IfcPresentationAppearanceResource/Entities/IfcTextureCoordinateIndicesWithVoids/Documentation.md]:
```suggestion
- the second dimension corresponds to the list position of the vertices at the n-th inner loop at _IfcIndexedPolygonalFaceWithVoids_
```

**Review by pjanck — CHANGES_REQUESTED** [review]
Two typos, otherwise seems fine.

**Comment by SergejMuhic** [comment]
Ping @jmirtsch @peterrdf @larswik @anborr @TLiebich @czapplitec one more review please. Thanks.

**Review by pjanck — APPROVED** [review]
lftm

**Review by TLiebich — APPROVED** [review]
looks fine

**Pull request merged**